api/.env

```diff
@@ -1,4 +1,3 @@
 POSTGRES_USER=postgres
-POSTGRES_HOST=127.0.0.1
```

Comment:
Ce n'est plus nécessaire? Pour quelle raison?

Reply:
@Anysiia 

Je l'ai fais passé [via le docker-compose](https://github.com/S-Stanley/ft_transcendence/blob/stan-add-all-services-to-docker-compose/docker-compose.yml#L21), etant donné qu'on ne peux pas reach le container db avec localhost, il faut faire passer l'ip.

L'ip etant dynamique et change a chaque demarrage docker s'occupe directement de convetir en faisant passé le nom du service (dans notre cas postgres).

[Plus d'info](https://stackoverflow.com/questions/43117429/how-to-pass-container-ip-as-env-to-other-container-in-docker-compose-file)

Pour faire un test tu peux run la commande
```bash
docker inspect ft_transcendence-postgres-1 | grep '"IPAddress":'
```

Si tu relance le container normalement l'ip aura legerement changé